app/src/main/res/layout/activity_main.xml

```diff
@@ -4,90 +4,76 @@
+    tools:context=".MainActivity">
 
     <TextView
         android:id="@+id/name"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
+        android:padding="15dp"
         android:text="Name:"
-        android:textStyle="bold"
         android:textSize="16sp"
-        android:padding="15dp"
+        android:textStyle="bold"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent"
-        />
+        app:layout_constraintTop_toTopOf="parent" />
 
     <TextView
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
+        android:padding="15dp"
         android:text="Email:"
-        android:textStyle="bold"
         android:textSize="16sp"
-        android:padding="15dp"
+        android:textStyle="bold"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/name"
-        />
+        app:layout_constraintTop_toBottomOf="@+id/name" />
 
     <EditText
         android:id="@+id/edit_name"
+        style="@style/Widget.AppCompat.Spinner.Underlined"
         android:layout_width="300dp"
         android:layout_height="wrap_content"
+        android:padding="15dp"
         app:layout_constraintLeft_toRightOf="@+id/name"
-        app:layout_constraintTop_toTopOf="parent"
-        style="@style/Widget.AppCompat.Spinner.Underlined"
-        android:padding="15dp"/>
+        app:layout_constraintTop_toTopOf="parent" />
 
     <EditText
         android:id="@+id/edit_email"
+        style="@style/Widget.AppCompat.Spinner.Underlined"
         android:layout_width="300dp"
         android:layout_height="wrap_content"
+        android:padding="15dp"
         app:layout_constraintLeft_toRightOf="@+id/name"
-        app:layout_constraintTop_toBottomOf="@+id/edit_name"
-        style="@style/Widget.AppCompat.Spinner.Underlined"
-        android:padding="15dp"/>
+        app:layout_constraintTop_toBottomOf="@+id/edit_name" />
 
     <Button
         android:id="@+id/button"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        app:layout_constraintTop_toBottomOf="@+id/edit_email"
+        android:backgroundTint="@color/lightBlue"
         android:text="ADD RECORD"
-        android:backgroundTint="@color/lightBlue"/>
+        app:layout_constraintTop_toBottomOf="@+id/edit_email" />
 
     <TextView
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        app:layout_constraintTop_toBottomOf="@id/button"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintEnd_toEndOf="parent"
         android:layout_marginTop="50dp"
-        />
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/button" />
+
 
-    <androidx.recyclerview.widget.RecyclerView
```

Comment:
I removed this recyclerview. You can add it back properly

Reply:
Why ? What was wrong with recyclerview?

Reply:
The textview was in the middle of it